src/hooks/__tests__/useCreateActivationKey.test.js

```diff
@@ -0,0 +1,38 @@
+import { renderHook } from '@testing-library/react-hooks';
+import fetch, { enableFetchMocks } from 'jest-fetch-mock';
+import { createQueryWrapper } from '../../utils/testHelpers';
+import useCreateActivationKey from '../useCreateActivationKey';
+import TestRenderer from 'react-test-renderer';
+
+enableFetchMocks();
+
+describe('useActivationKeys', () => {
+  const { act } = TestRenderer;
+  beforeEach(() => {
+    Object.defineProperty(window, 'insights', {
+      value: {
+        chrome: {
+          auth: {
+            getToken: jest.fn(),
+          },
+        },
+      },
+    });
+  });
+  it('creates activation key', async () => {
+    fetch.mockResponseOnce(JSON.stringify({ body: { id: 'id' } }));
+    const keyParams = {
+      name: 'A',
+      role: 'role',
+      serviceLevel: 'sla',
+      usage: 'usage',
+    };
+    const { result } = renderHook(() => useCreateActivationKey(), {
+      wrapper: createQueryWrapper(),
+    });
+    await act(async () => {
```

Comment:
This looks a bit odd. It might not be needed to await the act or pass it an async function, or is it?

Reply:
yes, In this case I need to wait for async callback, may be it's not a best way to test it, I am always open to suggestions.